bes_theme/assets/data/version_details/374-oneTBB-Versiondetails.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "version": "v2021.7.0",
+        "release_date": "",
```

Comment:
@sudhirverma Please provide the release date

Reply:
done